_pages/algorithm-development/ci-cd.md

```diff
@@ -127,3 +127,69 @@ To start using this in your own model development repositories, we have two full
 
 For more details on the configuration and capabilities of our GitHub Action, check out our detailed documentation on the GitHub Actions marketplace.
 <a href="https://github.com/algorithmiaio/algorithmia-modeldeployment-action" class="btn btn-default btn-primary"><i class="fa fa-github" aria-hidden="true"></i> TRY IT OUT: GitHub Actions for Model Training Repo</a>
+
+
+
+
+## GitLab CI/CD
+
+To deploy your algorithm files to Algorithmia everytime you push to your GitLab repository, make sure you have the following configured. 
+
+### Algorithm Files
+Similar to your default algorithm repo template, make sure your `src` folder and `requirements.txt` are in the root level of the repository. 
```

Comment:
```suggestion
Similar to your default algorithm-repository template, make sure your `src` folder and `requirements.txt` are in the root level of the repository. 
```

Reply:
i think this is clearer, otherwise it sounds like "default" refers to "algorithm" not "algorithm repository template"